xla/service/gpu/model/hlo_op_profiler_test.cc

```diff
@@ -48,6 +50,12 @@ TEST_F(HloOpProfilerTest, BasicMeasurementsAreCorrect) {
             1000);
 }
 
+TEST_F(HloOpProfilerTest, UnsupportedCombinationsDoNotCrash) {
+  HloOpProfiler profiler(test_runner_);
```

Comment:
This line causes a compile error:

```
note: candidate constructor not viable: cannot bind base class object of type 'HloRunnerInterface' to derived class reference 'HloRunner &' for 1st argument
   42 |   explicit HloOpProfiler(HloRunner& runner);
```

Reply:
was a conflict with https://github.com/openxla/xla/commit/5ad253ae69b2a43c91bda0c1a946c0a87c1f054f